### ISSUE TITLE
Add a link to the archival exception message

### ISSUE THREAD

**Issue by rhu001** [eScholarship/ucpec-static#53]
Please add a link to the archival exception message that is located under the top nav across all pages. "web accessibility" in second sentence. Link provided below.

This website contains archived content made available for reference, research, or recordkeeping purposes. These materials are not actively maintained and may not meet current [web accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/) standards, such as WCAG 2.1 Level AA. If you need assistance accessing this content or require an accessible format, [please contact us](https://cdlib.org/about/policies-and-guidelines/accessibility/). We are committed to ensuring access for all users and will provide accommodations promptly upon request.

**Comment by nicktahani** [comment]
Hi @rhu001, I've updated [one book page](https://publishing.cdlib.org/ucpressebooks/public/book/abuses.html) with the new language. Let me know if it looks good and I'll move forward with the remaining pages. Thanks!

**Comment by bibliotechy** [comment]
@nicktahani I'm just realizing that the Accessibility icon in the statement doesn't have an alt tag. It is decorative, so even just a alt="" woudl be fine.

**Comment by nicktahani** [comment]
@bibliotechy updated.

**Comment by rhu001** [comment]
Hi @nicktahani the new language looks good. Thanks! Good catch on the alt tag @bibliotechy !

**Comment by nicktahani** [comment]
@rhu001 @bibliotechy all remaining pages have been updated and are live. Thanks!
